examples/interactive_play.py

```diff
@@ -87,7 +87,12 @@ def step_env(env, action_name, action_args):
 
 
 def get_input_vel_ctlr(
-    skip_pygame, arm_action, env, not_block_input, agent_to_control
+    skip_pygame,
+    arm_action,
+    env,
+    not_block_input,
+    agent_to_control,
+    use_joint_control,
```

Comment:
it looks like this only works for humanoids. Is that correct? If so I would rename this argument.

Reply:
Resolved in last commit

Reply:
Hm I still see the variable name as `use_joint_control `. Is the change pushed?